src/components/Transaction.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import "./Components.css"
+import axios from 'axios';
+
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import Col from 'react-bootstrap/Col';
+import Card from 'react-bootstrap/Card';
+
+import HourGlass from '../img/hourglass.png';
+import Shaka from '../img/shaka.png'
+
+import useForm from './../hooks/Formulario';
+
+const Transaction = () =>{
+
+  const { form, onChange, resetForm } = useForm({
+    formFrom: '',
+    formTo: '',
+    formHours: '',
+    formWhat: '',
+  });
+
+  const submitForm = event => {
+    event.preventDefault()
+  }
+
+  
+  const handleChange = event => {
+    const { name, value } = event.target;
+    onChange(name, value);
+  };
+
+  const handleSubmit = async() =>{
+    const body = {
+      formFrom: form.formFrom,
```

Comment:
Eu tava pensando agora sobre padronizar nossa resposta :thinking: 
Poderia ser algo do tipo:
body = {
   transaction: Object.assign(form)
}
Onde form seria um objeto:
{
   from: <value>
   to: <value>,
   hours: <value>,
   what: <value>
}

Assim a gente já deixa o body da request com uma descrição de dados consistente e fácil de expandir.
E tira essa redundância de `form.form`.

Reply:
Acho que pode ser assim.

Reply:
Na verdade acho que seria melhor arrumar tanto a requisição quanto a resposta quando o endpoint ficar pronto. Assim já fazemos a versão "definitiva"